qt-models/maplocationmodel.cpp

```diff
@@ -10,6 +10,24 @@
 
 #define MIN_DISTANCE_BETWEEN_DIVE_SITES_M 50.0
 
+// MKW If "Map Short Names" preference is set, only return the last component
+// of the full dive site name.
+// Example:
+// Japan/Izu Peninsula/Atami/Chinsen-Aft
+//    Short name: Chinsen-Aft
+static QString siteMapDisplayName(const char *sitename)
+{
+	const char Separator = '/';
```

Comment:
we have tried to avoid imposing more syntax or a mandatory taxonomy on the location name.
This fixed separator isn't ideal, but it seems somewhat reasonable. The implied order also isn't ideal, but making this more flexible only makes things more complex, requires more options, and doesn't really significantly improve the user experience, so I think I'm ok with this.

Reply:
Hi Dirk, thank you for your comments.

Slash is already the baked-in separator used by the exporter (at least for divelog.de) As is the ordering (from most generic to most detailed). 

Prior to using Subsurface my personal preference was actually the other way around, and I used a different separator.

So these choices were made due to the existing Subsurface implementation.

As for it being optional, I'm sorry, I must have made a mistake, it was my intent that the default is "false". I'll make the relevant changes as soon as I'm back home (enjoying a short motorbike trip with friends for my birthday :)